test/unit_tests/backup/BackupRepositorySpec.js

```diff
@@ -19,6 +19,20 @@
 
 // grunt test_init && grunt test_run:backup/BackupRepository
 
+const noop = () => {};
+
+const conversationId = '35a9a89d-70dc-4d9e-88a2-4d8758458a6a';
+// prettier-ignore
+/* eslint-disable comma-spacing, key-spacing, sort-keys, quotes */
+const conversation = {"id": conversationId,"accessModes":["private"],"accessRole":"private","creator":"1ccd93e0-0f4b-4a73-b33f-05c464b88439","name":"Tom @ Staging","status":0,"team_id":null,"type":2,"others":["a7122859-3f16-4870-b7f2-5cbca5572ab2"],"last_event_timestamp":2,"last_server_timestamp":2,"archived_state":false,"archived_timestamp":0,"muted_state":false,"muted_timestamp":0};
```

Comment:
We have test data available in `payloads.js` which exposes it to the window namespace for this purpose. It might not be fully up to date with the latest fields but let's update and use that instead.

Reply:
Oh nice! Will do

Reply:
Talked with Gregor. It seems to be a lot of work to set the `payload.js` back on track (seems a little outdated)

Maybe something for the future